Resources/public/css/theme/views/subitem/listmore.css

```diff
@@ -10,3 +10,21 @@
 .ez-view-subitemlistmoreview .ez-subitemlist-no-sub-items {
     font-style: italic;
 }
+
+.ez-view-subitemlistmoreview .ez-subitem-column-sortable:hover:after {
+    content: "\E926";
+    font-family: 'ez-platformui-icomoon';
```

Comment:
instead of adding `font-family` (and maybe `vertical-align`) you should add the class `ez-font-icon` on `ez-subitem-column-sortable` element